django_select2/forms.py

```diff
@@ -98,21 +98,23 @@ def _get_media(self):
         .. Note:: For more information visit
             https://docs.djangoproject.com/en/1.8/topics/forms/media/#media-as-a-dynamic-property
         """
+        lang = get_language()
+        i18n_name = None
         try:
-            # Values in this dict had been removed from Django as language_code in favor of the keys
-            known_fallback_map = {
-                "zh-hans": "zh-cn",
-                "zh-hant": "zh-tw",
-            }
-
-            lang = get_language()
-            lang = known_fallback_map.get(lang, lang)
-
-            # get_language() will always return a lower case language code, where some files are named upper case.
-            i = [x.lower() for x in settings.SELECT2_I18N_AVAILABLE_LANGUAGES].index(lang)
-            i18n_file = ('%s/%s.js' % (settings.SELECT2_I18N_PATH, settings.SELECT2_I18N_AVAILABLE_LANGUAGES[i]), )
-        except ValueError:
-            i18n_file = ()
+            from django.contrib.admin.widgets import SELECT2_TRANSLATIONS
+            i18n_name = SELECT2_TRANSLATIONS.get(lang)
+            if i18n_name not in settings.SELECT2_I18N_AVAILABLE_LANGUAGES:
+                i18n_name = None
```

Comment:
I don't know whether this check is necessary. I guess currently it allow user to configure `SELECT2_I18N_AVAILABLE_LANGUAGES` ( for example, restrict i18n language allowed? I don't know.)?